core/domain/feedback_services_test.py

```diff
@@ -16,6 +16,7 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
```

Comment:
I think we should add a test that ensures that the new fields added here are actually cached? I don't see such a test.

Reply:
Done.